client/rest/test/plugins/rosetta/accountRoutes_spec.js

```diff
@@ -199,10 +238,217 @@ describe('account routes', () => {
 			skipBlockIdentifierTest: true
 		});
 
-		addAccountSuccessTests('/account/coins', {
-			createValidRequest,
-			createRosettaAmount: createRosettaCoin,
-			Response: AccountCoinsResponse
+		const incomingSignerPublicKey = '527068DA90B142D98D27FF9BA2103A54230E3C8FAC8529E804123D986CACDCC9';
+		const incomingRecipientAddress = 'TDI2ZPA7U72GHU2ZDP4C4J6T5YMFSLWEW4OZQKI';
+		const signerPublicKey = 'ED7FE5166BDC65D065667630B96362B3E57AFCA2B557B57E02022631C8C8F1A6';
+		const recipientAddress = 'TARZARAKDFNYFVFANAIAHCYUADHHZWT2WP2I7GI';
```

Comment:
imo naming is a bit awkward as (incomingSignerPublicKey, recipientAddress) and (incomingRecipientAddress, signerPublicKey) are paired.

might be better to use concepts like TargetPublicKey/Address and OtherPublicKey/Address where target matches accountid send to route